### ISSUE TITLE
CP window scrolls to top when Vue is done loading

### ISSUE THREAD

**Issue by robdekort** [statamic/cms#8290]
### Bug description

I'm noticing this behaviour on a site with lots of content blocks and fields. When you open an entry edit page in the CP you can see Vue is filling out all the data in the replicator. It's still loading everything. Users (including me) would probably go down, click on a block and start editing. A few seconds later - presumably when Vue has done it's thing - the whole page scrolls up which is confusing and a bit frustrating. 

https://github.com/statamic/cms/assets/69107412/bade9499-304e-444a-9297-313bbedda1b6

### How to reproduce

- Fill up a page with a bunch of replicator blocks + fields inside. 
- Load the page
- Scroll down
- Wait for Vue to be done
- Get scrolled back up

### Environment

```yaml
Environment
Laravel Version: 9.52.8
PHP Version: 8.2.3
Composer Version: 2.2.7
Environment: local
Debug Mode: ENABLED
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 7
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: 4.6.0 PRO

Statamic Addons
jacksleight/statamic-bard-mutator: 2.3.0
jonassiewertsen/statamic-livewire: 2.12.0
rias/statamic-color-swatches: 2.2.0
studio1902/statamic-peak-browser-appearance: 2.0
studio1902/statamic-peak-commands: 2.1
studio1902/statamic-peak-seo: 2.2
studio1902/statamic-peak-tools: 3.2
```

### Installation

Starter Kit using via CLI

### Antlers Parser

runtime (new)

**Comment by websmyth** [comment]
Additionally, Live Preview doesn’t work until all of these fields have loaded (the preview opens, but hangs on a blank page).

**Comment by robdekort** [comment]
> Additionally, Live Preview doesn’t work until all of these fields have loaded (the preview opens, but hangs on a blank page).

That probably deserves a new issue. But yeah it would be nice if the button would stay disabled until everything is loaded.

**Comment by robdekort** [comment]
This seems fixed by #8716 now the page seems to load differently, and faster.
